src/extension.ts

```diff
@@ -66,6 +67,8 @@ export async function activate(
   handleSelection(context);
   handleUninstall(() => uponUninstall(context));
 
+  registerCodeReview();
```

Comment:
this is too soon(you wont get the capabilities here ), the capabilities are fetched in the `backgroundInit`   https://github.com/codota/tabnine-vscode/blob/3b9e6fa06148544a4ef226b99a63dc64cbd7e57b/src/extension.ts#L100

Reply:
I'm checking the capabilities only on activeEditorChanged event